rdkit/Chem/fmcs/fmcs.py

```diff
@@ -674,24 +673,23 @@ def _check_atom_classes(molno, num_atoms, atom_classes):
 # prune. But so far I don't have a test set which drives the need for
 # that.
 
+# UPDATED by Ichiru Take: Use collections.Counter to count the bond types, which is much faster.
+# See here: https://stackoverflow.com/questions/44012479/intersection-of-two-counters
```

Comment:
You don't need to add this comment. It doesn't add anything

Reply:
OK, I got your idea! I would fix it in the next patch. Thank you for your review.